docs/source/reference/migration/from-router-v1.mdx

```diff
@@ -211,13 +232,112 @@ limits:
+- `apollo_authorization::scopes::required` -> `apollo::authorization::required_scopes`
+- `apollo_authorization::policies::required` -> `apollo::authorization::required_policies`
+- `apollo_operation_id` -> `apollo::supergraph::operation_id`
+- `apollo_override::unresolved_labels` -> `apollo::progressive_override::unresolved_labels`
+- `apollo_override::labels_to_override` -> `apollo::progressive_override::labels_to_override`
+- `apollo_router::supergraph::first_event` -> `apollo::supergraph::first_event`
+- `apollo_telemetry::client_name` -> `apollo::telemetry::client_name`
+- `apollo_telemetry::client_version` -> `apollo::telemetry::client_version`
+- `apollo_telemetry::studio::exclude` -> `apollo::telemetry::studio_exclude`
+- `apollo_telemetry::subgraph_ftv1` -> `apollo::telemetry::subgraph_ftv1`
+- `cost.actual` -> `apollo::demand_control::actual_cost`
+- `cost.estimated` -> `apollo::demand_control::estimated_cost`
+- `cost.result` -> `apollo::demand_control::result`
+- `cost.strategy` -> `apollo::demand_control::strategy`
+- `experimental::expose_query_plan.enabled` -> `apollo::expose_query_plan::enabled`
+- `experimental::expose_query_plan.formatted_plan` -> `apollo::expose_query_plan::formatted_plan`
+- `experimental::expose_query_plan.plan` -> `apollo::expose_query_plan::plan`
+- `operation_kind` -> `apollo::supergraph::operation_kind`
+- `operation_name` -> `apollo::supergraph::operation_name`
+- `persisted_query_hit` -> `apollo::apq::cache_hit`
+- `persisted_query_register` -> `apollo::apq::registered`
+
+### Jaeger Tracing
+
+`jaeger` exporter has been removed as Jaeger fully supports the OTLP format. To use the `otlp` exporter change your router config:
+
+```yaml title="router.yaml"
+telemetry:
+  exporters:
+    tracing:
+      propagation:
+        jaeger: true
+      otlp:
+        enabled: true
+```
+
+And ensure that you have enabled OTLP support in your Jaeger instance using `COLLECTOR_OTLP_ENABLED=true` and exposing ports `4317` and or `4318` for gRPC and HTTP respectively.
+
+### Enforce CORS configuration
+
+In Router 1.x, invalid values in the CORS configuration, such as malformed regexes, were ignored with an error logged.
+In Router 2.x, such values prevent router startup.
+
+If your Router 2.x fails to start with an error message like this:
+
+```
+could not create router: CORS configuration error:
+```
+
+Check the [CORS configuration documentation](../../routing/security/cors) and fix the problems or delete the invalid values.
+
+### Header propagation
+
+<!-- PR: https://github.com/apollographql/router/pull/6621 -->
+
+[Header propagation](/docs/graphos/routing/header-propagation#insert) lets you select data from a client request body using a JSON path, and pass it to subgraphs in a header.
+
+In 2.x, a spec-compliant [JSONPath](https://www.ietf.org/archive/id/draft-goessner-dispatch-jsonpath-00.html) path is used, so a `$` is now required to select the root element:
+
+```diff,yaml
```

Comment:
I hope our docs system supports this. I think diff syntax would be the best way to show certain config changes.

Reply:
Using normal yaml with comments instead for now.